test-apps/ui-test-app/src/frontend/appui/oidc/SignIn.tsx

```diff
@@ -12,7 +12,7 @@ import { FrontendAuthorizationClient, isFrontendAuthorizationClient } from "@ben
 import { IModelApp } from "@bentley/imodeljs-frontend";
 import { SignIn as SignInBase } from "@bentley/ui-components";
```

Comment:
We probably should've removed this from ui-components as well

Reply:
Agreed, @pbell97 mind removing this component too?